src/views/MainPage/MainPage.js

```diff
@@ -65,21 +58,83 @@ function renderMainView() {
   const creditsButton = Button('Credits', 'credits-button', true, 'click', handleNavigationButtonClick);
 
   navContainer.append(quizButton, leaderboardButton, adoptionButton, creditsButton);
-  const navItems = navContainer.querySelectorAll('a');
+}
+
+// Render Bubbles structure
+function renderBubbles() {
+  const bubbleFact = createFactBubble();
+  document.querySelector('.bubbles').append(bubbleFact);
+  const bubbleAdoption = createAdoptionBubble();
+  document.querySelector('.bubbles').append(bubbleAdoption);
+}
+
+function createFactBubble() {
+  const bubbleFact = Bubble('higher', true, 'Did you know?');
+  bubbleFact.classList.add('fact');
+  return bubbleFact;
+}
+
+function createAdoptionBubble() {
+  const bubbleAdoption = Bubble('lower', true, AdoptionBubbleContent.HEADER_TEXT, AdoptionBubbleContent.SENTENCES);
+  bubbleAdoption.classList.add('adoption');
+  return bubbleAdoption;
+}
+
+// Render Bubbles content
+function renderBubblesContent() {
+  renderAdoptionBubbleContent();
+  renderFactBubbleContent();
+}
 
-  navItems.forEach((item) => {
-    item.addEventListener('click', () => {
-      alert(`Hello! I'm ${item.dataset.name} site`);
-    });
-  });
+async function renderFactBubbleContent() {
+  const factObj = await getAnimalFact();
+  renderAnimalFact(factObj);
+  renderAnimalFactImg(factObj);
 }
 
-async function renderAnimalFact() {
-  const factItem = document.querySelector('.bubble-text');
+function renderAdoptionBubbleContent() {
+  renderAnimalAdoptionImg();
+}
+
+async function renderAnimalFact(factObj) {
+  const factSentenceItem = document.querySelector('.bubble.fact .bubble-text');
   const factSentence = document.createElement('p');
-  const FACT =
-    "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.";
 
-  factSentence.innerText = FACT;
-  factItem.append(factSentence);
+  factSentence.innerText = factObj.sentence;
+  factSentenceItem.append(factSentence);
+}
+
+function renderAnimalFactImg(factObj) {
+  const factImageItem = document.querySelector('.bubble.fact .bubble-img');
+  createImage(factImageItem, factObj);
+}
+
+function renderAnimalAdoptionImg() {
+  const adoptionImageItem = document.querySelector('.bubble.adoption .bubble-img');
+  createImage(adoptionImageItem, AdoptionBubbleContent.IMG);
+}
+
+function createImage(domElem, obj) {
+  if (domElem instanceof Element) {
+    const imgElem = document.createElement('img');
+    imgElem.src = obj.pathOrUrl;
+    imgElem.alt = obj.alt;
+    domElem.append(imgElem);
+  }
+}
```

Comment:
there is no need to check if domElem is instance of Element, because you use this function only on the elements